letsencrypt/crypto_util.py

```diff
@@ -171,6 +172,35 @@ def csr_matches_pubkey(csr, privkey):
         return False
 
 
+def import_csr_file(csrfile, contents):
+    """Import a CSR file, which can be either PEM or DER.
+
+    :param str csrfile: CSR filename
+    :param str contents: contens of the CSR file
+
+    :rtype: tuple
```

Comment:
The returns documentation can be cleaned up a bit:

1. Everywhere else, we put `:returns:` before `:rtype:`. I think we should continue doing this for consistence of our code and documentation.
2. The order of the elements in the `returns` tuple is wrong.
3. We get errors when building the docs due to this line:

```
OpenSSL FILETYPE_ representing DER or PEM,
```

Error was:
```
letsencrypt/crypto_util.py:docstring of letsencrypt.crypto_util.import_csr_file:8: ERROR: Unknown target name: "filetype".
```

Replacing that line with something like [this](https://github.com/letsencrypt/letsencrypt/pull/2684/files#diff-31d015f64b83c8ff85723c569cec870fR268) would fix the problem.